pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -260,7 +260,7 @@ func (o *Options) Run() error {
 							Image:   upgrade.Release.Image,
 						}
 						if o.AllowNotRecommended {
-							fmt.Fprintf(o.ErrOut, "warning: with --allow-not-recommended you have accepted the risks with %s and bypassing %s=%s %s: %s\n", o.To, c.Type, c.Status, c.Reason, c.Message)
+							fmt.Fprintf(o.ErrOut, "warning: with --allow-not-recommended you have bypassed %s=%s %s: %s and accepted the risks with %s\n", c.Type, c.Status, c.Reason, c.Message, o.To)
```

Comment:
I think you want to keep `Message` at the tail of this string, because it might run to multiple paragraphs, and by the time folks get around to `and accepted the risks with ${VERSION}`, they may have forgotten the initial line of the `warning`.